style.css

```diff
@@ -1 +1,218 @@
 /*  Add your styling here */
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed&display=swap');
+
+/* General Styles */
+body {
+    font-family: "Roboto Condensed", Roboto, sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    overflow-x: hidden;
+}
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+    color: #000;
+}
+
+h1, h2, h3, h4 {
```

Comment:
I don't think its a good idea to have the font weight of all the h tags the same, the idea of distingushing between h1-h6 is mainly through the difference in font-weight as well as size

Reply:
I see. The designed seemed better to me with headings set to the  same font-weight. Will avoid doing it.